addons/website_forum/views/website_forum.xml

```diff
@@ -686,437 +982,553 @@
-                                <t t-set="label">Delete</t>
-                                <t t-set="title">Delete</t>
-                                <t t-set="inDropdown" t-value="True"/>
-                                <t t-set="icon" t-value="'fa-trash-o text-primary'"/>
-                                <t t-set="classes" t-value="'comment_delete border-bottom'"/>
-                                <t t-set="karma" t-value="not object.can_unlink and object.karma_unlink or 0"/>
-                            </t>
-                            <t t-call="website_forum.link_button">
-                                <t t-set="url" t-value="'/forum/' + slug(forum) + '/post/' + slug(object) + '/comment/' + slug(message) +  '/convert_to_answer'"/>
-                                <t t-set="label">Convert as a answer</t>
-                                <t t-set="inDropdown" t-value="True"/>
-                                <t t-set="icon" t-value="'fa-magic text-primary'"/>
-                                <t t-set="karma" t-value="not object.can_comment_convert and object.karma_comment_convert or 0"/>
-                            </t>
+    <div class="o_wforum_post_comments_container ml-2 ml-md-5">
+        <div t-if="len(object.website_message_ids)" class="o_wforum_comments_count_header mb-2">
+            <div class="text-muted font-weight-bold small">
+                <span class="o_wforum_comments_count" t-esc="len(object.website_message_ids)"/>
+                <t t-if="len(object.website_message_ids) == 1">Comment</t>
+                <t t-else="">Comments</t>
+            </div>
+        </div>
+        <div class="css_editable_mode_hidden o_wforum_readable">
+            <form t-attf-id="comment#{ object._name.replace('.','') + '-' + str(object.id) }" class="collapse p-1 oe_comment_grey js_website_submit_form"
+                t-attf-action="/forum/#{slug(forum)}/post/#{slug(object)}/comment" method="POST">
+                <div class="shadow bg-white rounded px-3 pt-3 pb-4 mb-4">
+                    <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
+                    <input name="post_id" t-att-value="object.id" type="hidden" class="mt8"/>
+                    <div class="d-flex w-100">
+                        <img class="d-none d-md-inline-block rounded-circle o_forum_avatar mr-3" t-att-src="website.image_url(user, 'image_128', '40x40')" alt="Avatar"/>
+                        <div class="w-100">
+                            <textarea name="comment" class="form-control mb-2" placeholder="Comment this post..."/>
+                            <div>
+                                <button type="submit" class="btn btn-primary">Post comment</button>
+                                <a t-attf-href="#comment#{ question._name.replace('.','') + '-' + str(question.id) }" data-toggle="collapse" class="btn border">Discard</a>
```

Comment:
This button somehow does not work on my side, instead of discarding the current comment section, it shows the first comment section, I suppose a wrong href or target ?

Reply:
Can't replicate :/

Reply:
So hitting discard on the second comment section does not open/close the first one for you?

![image](https://user-images.githubusercontent.com/10338094/64614716-68230400-d3d9-11e9-92b6-42d13769d252.png)

Reply:
I fixed that problem on my branch